deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -9,9 +9,26 @@
 set -u -e
 
 # Capture the usual signals and exit from the script
-trap 'echo "SIGINT received, simply exiting..."; exit 0' SIGINT
-trap 'echo "SIGTERM received, simply exiting..."; exit 0' SIGTERM
-trap 'echo "SIGHUP received, simply exiting..."; exit 0' SIGHUP
+#trap 'echo "SIGINT received, simply exiting..."; exit 0' SIGINT
+#trap 'echo "SIGTERM received, simply exiting..."; exit 0' SIGTERM
+#trap 'echo "SIGHUP received, simply exiting..."; exit 0' SIGHUP
+
+function cleanup() {
```

Comment:
For readability, I think it's better to move this function to below the definition of HOST_CNI_NET_DIR and MOUNTED_CNI_NET_DIR variables.